apps/web/src/app/(dashboard)/documents/data-table.tsx

```diff
@@ -59,6 +60,11 @@ export const DocumentsDataTable = ({ results }: DocumentsDataTableProps) => {
             accessorKey: 'created',
             cell: ({ row }) => <LocaleDate date={row.getValue('created')} />,
           },
+          {
+            header: 'Actions',
+            accessorKey: 'actions',
```

Comment:
Just double check that this doesn't cause an issue since it's not a property on the object.

Reply:
Sure, I'll look into it

Reply:
I tested locally and everything is working so far, it doesn't cause an issue. You can perform tests to see yourself too